SPARQLWrapper/Wrapper.py

```diff
@@ -1226,7 +1238,9 @@ def _content_type_in_list(real, expected):
             """
             return True in [real.find(mime) != -1 for mime in expected]
 
-        if "content-type" in self.info():
+        if "content-type" not in self.info():
```

Comment:
This looks like it can go into a separate PR even though it is just changing order of conditions.

Reply:
934e334